test/e2e/helpers.js

```diff
@@ -269,7 +271,13 @@ async function withFixtures(options, testSuite) {
       await setSolanaWebsocketMocks();
     }
 
-    const { mockedEndpoint, getPrivacyReport } = await setupMocking(
+    // Decide between the regular setupMocking and the passThrough version
```

Comment:
mm is there a reason why instead of using mock server + passthrough, we can't pass a flag to not use mockttp at all? So we don't start the server neither route everything to it.
Maybe that would over-complicate things? Just wondering :thinking:

Reply:
Either way is fine. To disable the proxy completely would have required some more changes to our webdriver wrapper classes though.

And this way, we can use assertions in our tests to see which network calls are being made. Not necessary now, but potentially very useful for these "real server" tests in the future.